helm_deploy/values-dev.yaml

```diff
@@ -35,6 +35,7 @@ generic-service:
     SOC_API_URL: https://manage-soc-cases-api-dev.hmpps.service.justice.gov.uk
     TOKEN_VERIFICATION_API_URL: https://token-verification-api-dev.prison.service.justice.gov.uk
     WHEREABOUTS_ENDPOINT_URL: https://whereabouts-api-dev.service.justice.gov.uk
+    PRISONER_SEARCH_API_URL: https://prisoner-search-dev.prison.service.justice.gov.uk
```

Comment:
Would be good to add this to: https://github.com/ministryofjustice/hmpps-digital-prison-services/blob/main/readme/building_and_running.md?plain=1#L93